app/controllers/merchant/dashboard_controller.rb

```diff
@@ -2,6 +2,7 @@ class Merchant::DashboardController < Merchant::BaseController
   before_action :require_merchant
 
   def index
+    @merchant = Merchant.find(current_user.merchant_id)
```

Comment:
This makes sense currently - may need updates as we continue to update this Milestone